### DISCUSSION TITLE
Template vars inside PW manger templates.

### DISCUSSION THREAD

**Discussion by bionade24** [twpayne/chezmoi#2454] [Q&A] [answered]
Hello, I'd like to have something like `SOFTWARE_KEY="{{ (keepassxc "Software:  .chezmoi.hostname ").Password }}"` 

Is this currently possible? 

If not, would it be possible to implement?

**Comment by twpayne** [comment, marked as answer]
Yes, you can do:

```
SOFTWARE_KEY="{{ (keepassxc (printf "Software: %s" .chezmoi.hostname)).Password }}"
```
